manifests/openmanage.pp

```diff
@@ -72,10 +67,33 @@
         notify  => Service['dataeng'],
         require => Package['srvadmin-storageservices'],
       }
+
+      service { 'dataeng':
+        ensure    => $service_ensure,
+        hasstatus => true,
+        tag       => 'dell',
+      }
+
     }
 
     'Debian': {
       include ::dell::openmanage::debian
+
+      if $dell::openmanage::debian::ver >= 950 {
+        ['datamgrd', 'eventmgrd', 'snmpd'].each |$_service| {
+          service { "dsm_sa_${_service}":
+            ensure    => $service_ensure,
+            hasstatus => true,
+            tag       => 'dell',
+          }
+        }
```

Comment:
This could be achieved without a loop:
```suggestion
        service { [ 'dsm_sa_datamgrd', 'dsm_sa_eventmgrd', 'dsm_sa_snmpd' ]:
          ensure    => $service_ensure,
          hasstatus => true,
          tag       => 'dell',
        }
```